### DISCUSSION TITLE
EnsembleStat verification without climatology data

### DISCUSSION THREAD

**Discussion by BinbinZhou-NOAA** [dtcenter/METplus#1407] [Configuration] [answered]
If there is no climatology data but just  verify non climatology related ensembel scores in ECNT line type.  For example, the 24hr precip ensemble verification for GEFS has no precip climatology data but still want to verify its RMSE and Srpread.  I tested this buy not setting CLIMO MEAN and STDEV files in the conf file. The execution is  running successfully and the output stat files  are generated.  I am not for sure the ECNT line type data in those stat files are correct or not ?

Thanks! 

Binbin
EMC/NCEP

**Comment by CPKalb** [comment, marked as answer]
Hi Binbin:

If you are just looking at RMSE and spread, a climatology is not needed to compute these, so the numbers should be accurate.  Are you wanting to use other statistics?

Christina

**Reply by BinbinZhou-NOAA** [reply]
Christina,

  That's great!   All ensemble precip verification scores from ECNT  I want are not depend on climatology data.  

Thanks!

Binbin
